str/helper/mt_extractor.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+# pylint: disable=missing-function-docstring,no-member
```

Comment:
I'm not sure either of these are relevant, you have one function with a docstring

Reply:
removed